tsb/mp/main.tf

```diff
@@ -1,26 +1,25 @@
 data "terraform_remote_state" "infra" {
-  count   = length(local.k8s_regions)
   backend = "local"
   config = {
-    path = "../../infra/${var.tsb_mp["cloud"]}/terraform.tfstate.d/${var.tsb_mp["cloud"]}-${count.index}-${local.k8s_regions[count.index]}/terraform.tfstate"
+    path = "../../infra/${var.tsb_mp["cloud"]}/terraform.tfstate.d/${var.tsb_mp["cloud"]}-${var.tsb_mp["cluster_id"]}-${local.k8s_regions[tonumber(var.tsb_mp["cluster_id"])]}/terraform.tfstate"
```

Comment:
@nacx fyi, i have updated the section to the a single state reference of the target MP cluster, defined by tsb_mp variable, before it has a ```count.index``` that was pointing to the cluster_id of MP, which would be always a 0, so I switched to the single state with the reference to tsb_mp.cluster_id variable

Reply:
Yah, I saw that pattern and I have a local change fixing that in the infra folders as well. Since our Makefile invocations are always driven individually by the index, there is no need for the sound in an of them and we can reuse the pattern everywhere, I think. I'll submit a PR once the previous one is merged.